README.md

```diff
@@ -1,5 +1,5 @@
 # Adyen dotnet API Library
-[![Build Status](https://travis-ci.org/Adyen/adyen-dotnet-api-library.svg?branch=develop)](https://travis-ci.org/Adyen/adyen-dotnet-api-library)
+[![Build Status](https://travis-ci.org/Adyen/adyen-dotnet-api-library.svg?branch=develop)](https://travis-ci.org/Adyen/adyen-dotnet-api-library) [![nuget](https://img.shields.io/nuget/v/adyen.svg)](https://www.nuget.org/packages/adyen/)
```

Comment:
this image: https://img.shields.io/nuget/v/adyen.svg)](https://www.nuget.org/packages/adyen is resulting in:
nuget package not found

Reply:
I believe you copied the link wrong. You can check the readme file in the branch. There are two separate links
[nuget](https://www.nuget.org/packages/adyen/)
[Image](https://img.shields.io/nuget/v/adyen.svg)